### ISSUE TITLE
Accessibility Audit: Improper heading hierarchy on homepage

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#961]
**ENVIRONMENT**

macOS 11.6 | Chrome Version 95.0.4638.69 (Official Build) (x86_64) | Voiceover
Chrome OS Version 93.0.4577.51 (Official Build) beta (64-bit) | Chromevox
Windows 10 Enterprise | Chrome Version 96.0.4664.45 (Official Build) (64-bit) | JAWS
Windows 10 Enterprise | Chrome Version 96.0.4664.45 (Official Build) (64-bit) | NVDA
Windows 10 Enterprise | Firefox Version 92.0.2 (64-bit) | JAWS

**ISSUE**

Turn on the Screen Reader, and go to the [homepage](https://detroit-public-dev.netlify.app/). Navigate through the headings and observe that the Language section at the top is marked as heading level 2 and is placed above Heading Level 1 in the page.
Headings in the page should be in a hierarchical order. Heading level 1 should be at the top of the page. It is not ideal to mark 'languages' as a heading. Instead, Languages can be placed under a banner landmark. This same issue is observed on other pages on the site as well.

To replicate attached video on a Macbook Chrome, use Voiceover (Command + F5) and then use the 'Rotor' (Control + Option + U) to find the Headings category.

https://user-images.githubusercontent.com/394908/148606114-729a1f4b-cd36-41d5-af45-d926821e2e95.mov

**Comment by ColinBuyck** [comment]
No longer valid
